README.md

```diff
@@ -82,11 +83,9 @@ You can join the team by posting a comment to
 
 ### Security
 
-If you think you have discovered a new security issue with any StrongLoop
-package, **please do not report it on GitHub**. Instead, send an email to
-reachsl@us.ibm.com with a full description and steps to reproduce.
-
-See [SECURITY.md](SECURITY.md) for more details.
+If you think you have discovered a new security issue with any LoopBack 4
```

Comment:
Let's use just `LoopBack` (with no version number). I think the disclosure process is the same regardless of LB version affected.

```suggestion
If you think you have discovered a new security issue with any LoopBack
```